### PR TITLE
Feature#2/login

### PR DESCRIPTION
**Pull request by MrCano369x**
# Description
What did you implemented/Why did you implemented this:
 - we add the admin authentication using devise,
 - we add a logout button to the menu and 
 - we add design to the views.
 - issue: #2 

  ## Screenshots
login

### CODE REVIEW

File: app/views/devise/shared/_error_messages.html.erb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<% if resource.errors.any? %>
+  <div id="error_explanation">
+    <h2>
+      <%= I18n.t("errors.messages.not_saved",
+                 count: resource.errors.count,
+                 resource: resource.class.model_name.human.downcase)
+       %>
```
Comment (AlejandroCuriel) [inline app/views/devise/shared/_error_messages.html.erb:7]: can't you just use 1 line instead of 3?

File: config/routes.rb [hunk 2]
```diff
@@ -1,10 +1,12 @@
 Rails.application.routes.draw do
   resources :company_branches
   resources :employees
+
```
Comment (AlejandroCuriel) [inline config/routes.rb:4]: do you need this blank?

**Review by germanruzca — DISMISSED** [review]

File: app/models/user.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (AlejandroCuriel) [inline app/models/user.rb:5]: can it be done in one line?

**Review by juperameza — DISMISSED** [review]
LGTM :) ♥

**Review by AlejandroCuriel — CHANGES_REQUESTED** [review]
according to the good practices of the book "Clean Code" if the characters of a line are not so long, it is not necessary to take another

File: app/views/devise/shared/_error_messages.html.erb (hunk 1 above)
Comment (MrCano369x) [reply app/views/devise/shared/_error_messages.html.erb:7]: we will not make the change, the line would be very long 🖏

File: config/routes.rb (hunk 2 above)
Comment (MrCano369x) [reply config/routes.rb:4]: changed 🖎

File: app/models/user.rb (hunk 3 above)
Comment (MrCano369x) [reply app/models/user.rb:5]: changed 🖎

**Review by AlejandroCuriel — APPROVED** [review]
LGTM :rocket: :robot:

**Review by germanruzca — APPROVED** [review]
LGTM

**Review by Frcomparan — APPROVED** [review]
LGTM

**Pull request merged**